indico/modules/events/abstracts/blueprint.py

```diff
@@ -44,6 +44,7 @@
 # Book of Abstracts
 _bp.add_url_rule('/manage/abstracts/boa', 'manage_boa', boa.RHManageBOA, methods=('GET', 'POST'))
 _bp.add_url_rule('/book-of-abstracts.pdf', 'export_boa', boa.RHExportBOA)
+_bp.add_url_rule('/book-of-abstracts.zip', 'export_boa_tex', boa.RHExportBOATeX)
```

Comment:
I think this should be manager-only. There is no need a participant of the event needs to get a zip archive with the latex source code of the BoA. And by making it manager-only there is also no need to do any caching, so we don't need to complicate those functions to handle separate caches.

That way you could even use a BytesIO to write the zip file to instead of creating it on disk.